packages/SwingSet/src/lib/message.js

```diff
@@ -72,7 +72,9 @@ export function insistVatDeliveryObject(vdo) {
       break;
     }
     case 'stopVat': {
-      assert(rest.length === 0);
+      assert(rest.length === 1);
+      const [disconnectObject] = rest;
```

Comment:
to be consistent, let's call this `disconnectObjectCapData` too

Reply:
I was going to do that but I was following the pattern of the parallel `startVat` request, which takes a `vatParameters` arg that's also capdata.  But since we're using `disconnectObjectCapData` everywhere it makes sense to do it here too.  (And I guess we're using `vatParameters` everywhere instead of, say `vatParametersCapData`, so...)